src/types/index.d.ts

```diff
@@ -21,6 +21,7 @@ export interface JestHTMLReporterConfiguration {
   dateFormat: string;
   executionTimeWarningThreshold: number;
   includeConsoleLog: boolean;
+  includeConsoleLogOrigin: boolean;
```

Comment:
The configuration needs also to be added to `./src/extractConfiguration.ts`: both to the `defaultValues` and the `typeParsers`